plugins/woocommerce-admin/client/dashboard/components/connect/index.js

```diff
@@ -157,20 +157,24 @@ Connect.defaultProps = {
 
 export default compose(
 	withSelect( ( select, props ) => {
-		const { getJetpackConnectUrl, isPluginsRequesting, getPluginsError } =
-			select( PLUGINS_STORE_NAME );
+		const {
+			getJetpackAuthUrl,
+			isOnboardingRequesting,
+			getOnboardingError,
+		} = select( ONBOARDING_STORE_NAME );
 
 		const queryArgs = {
-			redirect_url: props.redirectUrl || window.location.href,
+			redirectUrl: props.redirectUrl || window.location.href,
+			from: 'woocommerce-services',
 		};
-		const isRequesting = isPluginsRequesting( 'getJetpackConnectUrl' );
-		const error = getPluginsError( 'getJetpackConnectUrl' ) || '';
-		const jetpackConnectUrl = getJetpackConnectUrl( queryArgs );
+		const isRequesting = isOnboardingRequesting( 'getJetpackAuthUrl' );
+		const error = getOnboardingError( 'getJetpackAuthUrl' ) || '';
+		const jetpackAuthUrl = getJetpackAuthUrl( queryArgs ).url;
```

Comment:
This was a really smooth transition but I noticed that the plugins data store returned a string for `jetpackConnectUrl`, whereas the onboarding data store returns an object informing of the error, etc., so we have to extract the `.url` prop here.

I am wondering if it is OK to ignore the other fields in the response object indicating request success etc. provided we check `isRequesting` and `error`?